doc/release-process.md

```diff
@@ -111,16 +111,16 @@ The gbuild invocations below <b>DO NOT DO THIS</b> by default.
 ### Build and sign Bitcoin Core for Linux, Windows, and OS X:
 
     pushd ./gitian-builder
-    ./bin/gbuild --memory 3000 --commit bitcoin=v${VERSION} ../bitcoin/contrib/gitian-descriptors/gitian-linux.yml
+    ./bin/gbuild --num-make `nproc` --memory 3000 --commit bitcoin=v${VERSION} ../bitcoin/contrib/gitian-descriptors/gitian-linux.yml
```

Comment:
You could just provide the default of `--num-make 2` here, which works with `--memory 3000`
Doubling one of the numbers means you have to double the other one too, at least for KVM, for LXC the memory limit is  ignored which is why you didn't notice anything.